src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1750,6 +1750,13 @@ private CSDiagnosticInfo NotFound(CSharpSyntaxNode where, string simpleName, int
             bool encounteredForwardingCycle;
             string fullName;
 
+            // for attributes, suggest both, but not for verbatim name
+            if (options == LookupOptions.AttributeTypeOnly)
```

Comment:
LookupOptions is a combination of flags, use LookupOptions.IsAttributeTypeLookup extension method to check if we are looking for an attribute type. Use LookupOptions.IsVerbatimNameAttributeTypeLookup to check if attribute type name lookup is verbatim.